plasmapy/formulary/parameters.py

```diff
@@ -612,44 +748,102 @@ def thermal_speed(
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"rms"``                 | .. math::     | .. math::     | .. math::     |
+       |                           |    1          |    \sqrt{2}   |    \sqrt{3}   |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"mean_magnitude"``      | .. math::     | .. math::     | .. math::     |
+       |                           |    \sqrt{2/π} |    \sqrt{π/2} |    \sqrt{8/π} |
+       +--------------+------------+---------------+---------------+---------------+
+       | ``"nrl"``                 | .. math::                                     |
+       |                           |    1                                          |
+       +--------------+------------+---------------+---------------+---------------+
+
+    The coefficents can be directly retrieved using
+    `~plasmapy.formulary.parameters.thermal_speed_coefficients`.
+
+        .. rubric:: The Methods
+
+        In the following discussion the Maxwellian distribution
+        :math:`f(\mathbf{v})` is assumed to be 3D, but similar expressions can
+        be given for 1D and 2D.
+
+        - **Most Probable** ``method = "most_probable"``
+
+          This method expresses the thermal speed of the distribution by expressing
+          it as the most probable speed a particle in the distribution may have.
+          To do this we first define another function :math:`g(v)` given by
+
+          .. math::
+             \int_{0}^{\infty} g(v) dv
+                = \int_{-\infty}^{\infty} f(\mathbf{v}) d^3\mathbf{v}
+                \quad \rightarrow \quad
+                g(v) = 4 \pi v^2 f(v)
+
+          then
+
+          .. math::
+             g^{\prime}(v_{th}) = \left.\frac{dg}{dv}\right|_{v_{th}} = 0\\
+             \implies v_{th} = \sqrt{\frac{2 k_B T}{m}}
+
+        - **Root Mean Square** ``method = "rms"``
+
+          This method uses the root mean square to calculate an expression for
+          the thermal speed of the particle distribution, which is given by
+          defines the thermal speed as
+
+          .. math::
+             v_{th} = \left[\int v^2 f(\mathbf{v}) d^3 \mathbf{v}\right]^{1/2}
+                        = \sqrt{\frac{3 k_B T}{m}}
+
+        - **Mean Magnitude** ``method = "mean_magnitude"``
+
+          This method uses the mean speed of the particle distribution to
+          calcuate an expression for the thermal speed, which is given by
+
+          .. math::
+             v_{th} = \int |\mathbf{v}| f(\mathbf{v}) d^3 \mathbf{v}
+                         = \sqrt{\frac{8 k_B T}{\pi m}}
+
+
+        - **NRL Formulary** ``method = "nrl"``
+
+          The `NRL Plasma Formulary
+          <https://www.nrl.navy.mil/ppd/content/nrl-plasma-formulary>`_
+          uses the square root of the Normal distribution's variance
+          :math:`\sigma^2` as the expression for thermal speed.
```

Comment:
```suggestion
          as the expression for thermal speed.
```
[optional] We could possibly drop this if we don't use σ or σ² elsewhere.